root/static/scripts/common/components/CommonsImage.js

```diff
@@ -14,7 +14,7 @@ import entityHref from '../utility/entityHref.js';
 
 type Props = {
   +cachedImage: ?CommonsImageT,
-  +entity: CoreEntityT,
+  +entity: NonUrlRelatableEntityT,
```

Comment:
Why do we even keep this code around?

Reply:
I guess that the hope is that Commons will eventually overcome the issue that led us to disable it on our website.  It can be still used by local mirrors.